_includes/master/reqs-sys.md

```diff
@@ -15,17 +19,26 @@
   - Ubuntu 16.04
   - CentOS 7
   {% endif %}
-{% if include.orch == "Kubernetes" %}
-<br>
-- No other pod networking configurations stored in `/etc/cni/net.d`.
-{% endif %}
+
 
 ## Key/value store
 
-{{site.prodname}} {{page.version}} requires an [etcdv3 or later server](https://github.com/coreos/etcd/releases) accessible by all nodes in the cluster. 
-{% if include.orch == "Kubernetes" %} You can configure {{site.prodname}} to 
-access etcd directly or via the Kubernetes API.{% endif %}{% if include.orch == "OpenShift" %}
-Your nodes must access etcd directly, not via the Kubernetes API.{% endif %}
-{% if include.orch == "OpenStack" %}If you don't already have an etcd server
+{{site.prodname}} {{page.version}} requires a key/value store accessible by all 
+{{site.prodname}} components. {% if include.orch == "Kubernetes" %} On Kubernetes,
+you can configure {{site.prodname}} to access an etcdv3 server directly or to
+use the Kubernetes API datastore.{% endif %}{% if include.orch == "OpenShift" %} On
+OpenShift, {{site.prodname}} can share an etcdv3 key/value store wit OpenShift, or
+you can set up an etcdv3 key/value store dedicated to {{site.prodname}}.{% endif %}
+{% if include.orch == "OpenStack" %}If you don't already have an etcdv3 server
 to connect to, we provide instructions in the [installation documentation](../installation/).{% endif %}
 
+
+## Network requirements
+
+BGP traffic over TCP must be allowed. By default, these connections use port 179.
+
+If running with IPIP encapsulation (the default mode), IPIP traffic must also be
+allowed on all ports.
+
+> **Tip**: On GCE, you can set these as firewall rules. In AWS, use security groups.
```

Comment:
I'd reword this tip ever so slightly, I think this is a more natural phrasing:

"On GCE, you can allow this traffic using firewall rules. On AWS, use EC2 security group rules"